tests/lac/solver.cc

```diff
@@ -93,6 +94,8 @@ int main()
   SolverBicgstab<> bicgstab(control, mem);
   SolverRichardson<> rich(control, mem);
   SolverQMRS<> qmrs(control, mem);
+  SolverFIRE<>::AdditionalData fire_data(0.1, 1, 1);
```

Comment:
are these values different from the defaults? If not, no need to have the `fire_data` object. But if they are different, then why use values here that are different from the defaults?